### DISCUSSION TITLE
Telemetry with Prometheus and Grafana

### DISCUSSION THREAD

**Discussion by mcvoigt** [drakkan/sftpgo#326] [Ideas]
Hi
I just want to share, that today I installed Grafana to work together with Prometheus and the features that Grafana allows to perform are fantastic. For those who want to venture into installing Grafana and Prometheus follow the link.

https://prometheus.io/docs/visualization/grafana/
https://grafana.com/grafana/download

Previously, Nicola "drakkan master" had shared how to install and set Prometheus, but I ended up going a bit further and I liked the result combined with Grafana.
One of the features that I really liked about Grafana is the fact that it allows native autorefresh, more beautiful and personalized graphics and greater control over accessing the tool, allowing you to create users with passwords to access these graphics, etc.

About Prometheus and telemetry
https://github.com/drakkan/sftpgo/issues/304

Best Regards,
Michael Voigt

**Comment by drakkan** [comment]
Great! Thanks for sharing
